client/my-sites/plugins/minimum-jet-pack-version-notice.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import notices from 'notices';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	getSite,
+	isJetpackSite,
+	siteHasMinimumJetpackVersion
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends React.Component {
```

Comment:
You can `import React, { Component, PropTypes } from 'react'` and use these as shorthands, omitting `React.` where possible.